db/migrate/20220115171649_add_matched_to_payments.rb

```diff
@@ -0,0 +1,5 @@
+class AddMatchedToPayments < ActiveRecord::Migration
+  def change
+    add_column :payments, :matched, :boolean, default: false
```

Comment:
Please put the matched field on another model and use a has_one reference from Payment to the table and a belongs_to from the table to Payment. Adding fields to a model is not good practice and it breaks all of the tests